documentation/source/_templates/layout.html

```diff
@@ -0,0 +1,12 @@
+{% extends "!layout.html" %}
+
+{% block content %}
```

Comment:
Tempting to use `document` instead of `content` so this ends up below the breadcrumb